tests/library/gemm_test.py

```diff
@@ -130,12 +117,9 @@ def params_generator(grid):
     for implementation in Gemm.implementations:
         print("Testing {}...".format(implementation))
         try:
-            if args["grid"]:
-                for params in params_generator(param_grid):
-                    print("testing:", params)
-                    run_test(implementation, params)
-            else:
-                run_test(implementation, args)
+            for params in params_generator(param_grid):
```

Comment:
This takes a while to run currently (<5 min), hopefully this doesn't spam your CI. Depending on your opinion, we can scale this back a bit (although it did help me catch the weird bug with complex).

Reply:
@orausch That's too many options (good for an `*_exhaustivetest.py`). Please choose random {alpha,beta}, one size (23x24x25 or 10x11x12 are good because they're rectangular), and maybe run all the options in transA/transB. This will end up with four reasonable tests.